docs/available_templates/cli_python.rst

```diff
@@ -75,11 +75,10 @@ Design
     ├── pyproject.toml
     ├── README.rst
     ├── .readthedocs.yml
-    ├── src
-    │   └── project_name
-    │       ├── __init__.py
-    │       ├── __main__.py
-    │       └── py.typed
+    │  └── project_name
+    │      ├── __init__.py
+    │      ├── __main__.py
+    │      └── py.typed
```

Comment:
do we need this file? since its empty , I'd suggest removing it

Reply:
We need it. 

"If you would like to publish a library package to a package repository (e.g. PyPI) for either internal or external use in type checking, packages that supply type information via type comments or annotations in the code should put a py.typed file in their package directory. For example, with a directory structure as follows"